benchmarks/Benchmarks/CertificatesBenchmarks/Benchmarks.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Benchmark
+import Sources
+
+let benchmarks = {
+    Benchmark("Verifier", configuration: .init(warmupIterations: 1)) { benchmark in
+        await verifier()
+    }
+    
+    let runParseWebPKIRoots = parseWebPKIRoots()
+    Benchmark("Parse WebPKI Roots") { benchmark in
+        runParseWebPKIRoots()
+    }
+    
+    Benchmark("TinyArray non-allocating functions") { benchmark in
+        tinyArrayNonAllocationFunctions()
+    }
+    
+    Benchmark("TinyArray.append(_:)") { benchmark in
+        tinyArrayAppend()
+    }
```

Comment:
This uses the default metrics now, we should either use `all` or select the metrics we care about.

Reply:
I'd suggest you set up 

```swift
Benchmark.defaultConfiguration = .init(...)
```

at the top of the suite with whatever default config you want to use for all benchmarks (if the same), then you can easily add metrics for the whole suite without resorting to ".all" which may be overkill for most uses.

If you have subsets that needs another config just do:
```swift
    let someCustomConfiguration: Benchmark.Configuration = .init(...)

    Benchmark("xxx 1", configuration: someCustomConfiguration) { _ in
    }

    Benchmark("xxx 2", configuration: someCustomConfiguration) { _ in
    }
```

Also convenient for setting up thresholds.